utils/utils.jsx

```diff
@@ -1365,6 +1365,14 @@ export function imageURLForUser(userIdOrObject) {
     return Client4.getUsersRoute() + '/' + userIdOrObject.id + '/image?_=' + (userIdOrObject.last_picture_update || 0);
 }
 
+export function defaultImageURLForUser(userIdOrObject) {
```

Comment:
Why not only accept `userId` as parameter?

Reply:
The reason behind that was to mimic the imageURLForUser, which is used in a lot of places in the application, but this is very concrete, and I can use directly the id, so I going to change it.

Thanks, good catch :)

Reply:
Fixed :)